config/locales/ch_be.yml

```diff
@@ -81,6 +81,8 @@ ch_be:
       weak_password: 'Um d Sicherheit z verbessere, söttisch es komplizierteres Passwort wähle'
     teams:
       not_existing: 'Es existiert kes Team mit dr Id %{id}'
+    folders:
+      duplicated_inbox: 'Es cha nume ei inbox ordner im personal team gä'
```

Comment:
Maybe rename this to "Persönleche Team" since it is translated, but maybe this also just causes confusion, @mtnstar what would you say about this?
If it should stay in english maybe type it capitatlized, because it's swissgerman so maybe something like "Personal Team"

Reply:
@mtnstar said that you can use what you think works better either "Persönleche Team" or "Personal Team"